kubernetes/configmap/nginx.yaml

```diff
@@ -186,6 +186,17 @@ data:
                     }
                 }
             }
+        
+            # cdn headers
+            location ~ /$ {
```

Comment:
Does this mean only the location ending with a slash will be filtered?

The default setting of the SSP does not use trailing slashes in URL 🤔  

Also, I have several more questions about this solution:
- the location catches all urls, but serves only @storefront location, is that ok?
- do I assume correctly, headers are automatically added by the CDN?